_posts/2015-10-07-code-talks-2015-retrospective.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "code.talks 2015 retrospective"
+date: "2015-10-07 09:00:00"
+icon: comments
+categories: conferences
+authors: ["Jens"]
+---
+
+For the fifth time about 1.500 developers gathered for the yearly `code.talks` conference in Hamburg:
+on September 29 and 30 also a dozen ePagees went to the Cinemaxx Hamburg Dammtor to enjoy two days fully packed with [lots of talks](https://www.codetalks.de/2015/programm) from 13 different tracks.
+Each track focussed on a specific theme like E-Commerce, IT Management, Startups, New Technology, Big Data, Mobile, DevOps, Scaling, Infrastructure, UX/Frontend, PHP, JavaScript and Java - the latter also being sponsored by ePages.
+Each hour talks from these tracks where held in parallel at eight movie theater rooms.
+Watching these talks on huge cinema screens while sitting in cozy chairs and eating popcorn with nachos is a very special experience!
+
+Due to the huge amount of different talks there was almost always at least one interesting session for everyone - more often than not it was a tough choice between two or more competing topics being presented at the same time.
+Between the talks there was time for meeting and networking with other visitors to exchange impressions and ideas.
+All the time there was more than enough excellent food and drinks provided; not only water, coffee and tea, but of course also [1337 mate](http://www.1337mate.com/) ;-)
+
+Those not already too exhausted by the intense first day had the chance to spend their energy during an exclusive party in the evening/night, also with vouchers for free drinks.
+At next day's breakfast you could clearly tell who had a tough night, but judging from the full conference rooms only a few visitors slept in.
+
+## The talks
+
+I started the conference attending talks highly related to my current assignment at ePages:
+[Microservices](https://www.codetalks.de/2015/programm/radical-agility-with-autonomous-teams-and-microservices-in-the-cloud), [Docker containers](https://www.codetalks.de/2015/programm/patterns-in-a-containerized-world), [Consul](https://www.codetalks.de/2015/programm/die-cloud-im-griff-mit-consul), [Mesos & Marathon](https://www.codetalks.de/2015/programm/scalable-micro-services-with-apache-mesos-and-marathon) are technologies my team evaluates to create a new architecture for our product.
```

Comment:
Please don't mention in the blog directly that we are creating a new architecture. Rather describe it a bit more general.